modules/app/components/ResourceBox.tsx

```diff
@@ -72,23 +75,23 @@ const resources: Record<ResourceType, Resource> = {
     links: [
       {
         linkTitle: 'How to participate in MakerDAO governance?',
-        url: 'https://makerdao.world/en/learn/governance/how-voting-works/'
+        url: 'https://manual.makerdao.com/governance/voting-in-makerdao'
       },
       {
         linkTitle: 'What are Executive Votes?',
-        url: 'https://makerdao.world/en/learn/governance/how-voting-works#executive-votes'
+        url: 'https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance#executive-votes'
       },
       {
         linkTitle: 'How to manually verify Executive Spells?',
-        url: 'https://makerdao.world/en/learn/governance/audit-exec-spells'
+        url: 'https://manual.makerdao.com/governance/verification/executive-audit'
       },
       {
         linkTitle: 'How is voting weight calculated?',
-        url: 'https://makerdao.world/en/learn/governance/how-voting-works#governance-polls-and-executive-votes'
+        url: 'https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance#how-is-the-voting-calculated'
       },
```

Comment:
This one looks duplicated as well

Reply:
One of the links goes in the polling page and the other in the executive page, should we create another object for common links?

Reply:
I see now. We could but not a big deal, I think it's fine as is.